src/containers/barray.mli

```diff
@@ -76,24 +76,23 @@ val mapi : (int -> 'a -> 'b) -> 'a array -> 'b array
     function is applied to the index of the element as first argument,
     and the element itself as second argument. *)
 
-val fold_left : ('a -> 'b -> 'a) -> 'a -> 'b array -> 'a
+val foldLeft : ('a -> 'b -> 'a) -> 'a -> 'b array -> 'a
```

Comment:
Can we use the same terminology as Immutable.re (reduce, reduceReversed). I had a conversation with @jordwalke and he made a compelling argument against using left/right.